src/azul/plugins/metadata/hca/indexer/aggregate.py

```diff
@@ -123,14 +123,21 @@ class CellSuspensionAggregator(GroupingAggregator):
     def _transform_entity(self, entity: JSON) -> JSON:
         return {
             **entity,
-            'total_estimated_cells': (entity['document_id'], entity['total_estimated_cells']),
+            'total_estimated_cells': (
+                entity['document_id'],
+                entity['total_estimated_cells']
+            ),
+            'total_estimated_cells_unadjusted': (
+                entity['document_id'],
+                entity['total_estimated_cells_unadjusted']
+            ),
         }
 
     def _group_keys(self, entity) -> tuple[Any, ...]:
         return frozenset(entity['organ']),
 
     def _get_accumulator(self, field) -> Optional[Accumulator]:
-        if field == 'total_estimated_cells':
+        if field in ['total_estimated_cells', 'total_estimated_cells_unadjusted']:
```

Comment:
```suggestion
        if field in {'total_estimated_cells', 'total_estimated_cells_unadjusted'}:
```

See https://stackoverflow.com/a/25368371/4171119